evergreen/config.yml

```diff
@@ -867,6 +889,23 @@ buildvariants:
     distros:
     - rhel70-large
 
+- name: ubuntu-valgrind
+  display_name: "Ubuntu 2204 Valgrind"
+  run_on: ubuntu2204-large
+  expansions:
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.20.3-linux-x86_64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    fetch_missing_dependencies: On
+    enable_valgrind: On
+    cmake_build_type: RelWithDebInfo
+    curl: "/opt/mongodbtoolchain/v3/bin/curl"
```

Comment:
is curl not just baked into the OS on ubuntu2204?

Reply:
It looks like it is:
```
ubuntu@ip-10-128-110-235:~$ which curl
/usr/bin/curl
ubuntu@ip-10-128-110-235:~$ curl --version
curl 7.81.0 (x86_64-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.13
Release-Date: 2022-01-05
```